.github/workflows/build.yml

```diff
@@ -119,74 +119,70 @@ jobs:
     needs: [ build, changes ]
```

Comment:
NOTE:
TBD: Some change to pass the correct preview endpoint url for the route service.